python/mit/binding.py.in

```diff
@@ -41,6 +41,7 @@ def errcheck(code_to_message):
     '''
     require_match = None in code_to_message.values()
     def callback(result, _func, _args):
+        result = int(result)
         if result in code_to_message:
```

Comment:
The call to `int()` might be unnecessary if `c_int128.__eq__()` is value equality.

Better still, `result = result.value`.

Reply:
That seems not to work for built-in ctypes types, which produce an int here somehow.